deployment/clouddeploy/gke-workers/environments/oss-vdb-test/backup.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: backup
+spec:
+  schedule: "0 18 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: cron
+            image: cron
+            imagePullPolicy: Always
+            env:
+            - name: GOOGLE_CLOUD_PROJECT
+              value: "oss-vdb-test"
+            - name: BACKUP_BUCKET
+              value: "osv-test-backup"
+            - name: ENTRYPOINT
+              value: "./backup/backup.py"
```

Comment:
I think this needs to be set as the [command](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/) instead.

Also, you copy backup into `/usr/local/bin/` in the Dockerfile, while I don't think is the workdir of the container, so I don't think using the local path works as-is.
Maybe change the Dockerfile to copy the code into `./` instead?

Reply:
Thanks Michael! I still kept python files under `usr/local/bin` but updated file path here.